### ISSUE TITLE
Bug: Unable to login

### ISSUE THREAD

**Issue by nicobarretoOC** [opencollective/opencollective#569]
Barbara from smartbear is trying to cancel a subscription, but every time she logs in she cant access the menu to edit subscriptions:
![image001](https://user-images.githubusercontent.com/30665166/32058971-55a92d64-ba30-11e7-91a4-6d550d0bf7ea.jpg)

***
Thank you for taking the time to report an issue 🙏

The easier it is for us to reproduce it, the faster we can solve it.
So please try to be as complete as possible when filing your issue.

***

URL: 
Logged in as: (just provide your username or the url of your profile)
Error message if any:
Expected result: 
Browser:

Bonus point if you can add a screenshot :-)

Thank you and have a wonderful day/evening!
(and sorry for inconvenience! We'll do our best to solve this ASAP)

**Comment by xdamman** [comment]
Couldn't reproduce 
<img width="482" alt="screen shot 2017-10-27 at 6 30 29 pm" src="https://user-images.githubusercontent.com/74358/32097594-033430ea-bb45-11e7-882b-d6caf49489d2.png">

Please make sure you answers most of the questions in the new issue template (e.g. what browser was she using? Any error message in the console? (only if technical person, otherwise it's hard to ask for such information to a regular user), ...

**Comment by piamancini** [comment]
@xdamman here: 
![image001](https://user-images.githubusercontent.com/3671070/32103795-d3c0670a-bb19-11e7-9cd1-cb31e0fe5093.jpg)
she keeps seeing the login as loading and can't login

**Comment by nicobarretoOC** [comment]
Seems like Barbara cant login to edit her subscriptions...

Previous mail:

> Hey Barbara! We were unable to reproduce the error.
> 
> Any ways, I wanted to follow up with you to see if you were able to access the link to cancel your subscriptions here:
> http://opencollective.com/subscriptions
> 
> Let us know if you had any problems. (...)

Barbara's response:

> Nope, same issues, it asks me to login and then I get the email and zip, back to the beginning. Can you please cancel for me and send acknowledgment?

**Comment by piamancini** [comment]
@xdamman more update on this: the system acknowledges there's a user but still prompts to create one instead of sensing the email. We need to cancel her subscription before the 11/1, she's been trying to do it for a while: 
<img width="1189" alt="screen shot 2017-10-28 at 12 32 52 pm" src="https://user-images.githubusercontent.com/3671070/32134027-564a7004-bbdc-11e7-863a-6286eb5f829b.png">

**Comment by piamancini** [comment]
ok, managed to find an old email from her and cancel the subscription. 
but the bug is still happening.

**Comment by piamancini** [comment]
removing the high priority level since I unblocked user. @xdamman do you want to look into it?

**Comment by xdamman** [comment]
The problem is that somehow there are 163 collectives that don't have a slug (they are all of type USER and don't have a `createdAt` neither. Not sure what happened.

**Comment by xdamman** [comment]
It looks like they all also have an organization (e.g. rackspace, belcube, ...)
They were created after 2017-10-30 13:41:18.667+00
Maybe related to the host migration? @asood123

**Comment by xdamman** [comment]
It's basically users that have created an account for their organization and then have created an organization (or we created a collective for them @piamancini ?)

I've just updated it manually for Barbara. I changed the name of her User Collective to her personal name instead of cross browser testing, added a slug and createdAt (and twitterHandle / website / description in the process)

**Comment by piamancini** [comment]
hmmm weird. I haven't manually created a collective for them. 
we did the migration yesterday but this issue has been reported a while back. 
Let's look into it together next week

**Comment by nicobarretoOC** [comment]
Adding to this:

> Hey Pia and Nick, 
> 
> I'm trying to log into OC to perform some end of the month entries and I'm receiving an error message because I'm not a host of the open collective? I've also tried to log in under finance@*** but I haven't received a log in email. 
> 
> ​Any guidance would be helpful. ​
> 
> Best,
> Brittani

**Comment by piamancini** [comment]
@nicobarretoOC her host address is oc@** not finance@ (that said she should be receiving them email, but it's probably related to their setup)

**Comment by piamancini** [comment]
close we fixed this.
